extensions/sql-migration/src/dialog/assessment/assessmentUtils.ts

```diff
@@ -0,0 +1,216 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MigrationTargetType } from '../../api/utils';
+import { IssueCategory } from '../../constants/helper';
+import { Page, SavedInfo } from '../../models/stateMachine';
+
+export function parseAssessmentReport(assessmentReport: any): any {
+	const saveInfo: SavedInfo = {
+		xEventsFilesFolderPath: null,
+		closedPage: Page.ImportAssessment,
+		databaseAssessment: [],
+		databaseList: [],
+		databaseInfoList: [],
+		migrationTargetType: null,
+		azureAccount: null,
+		azureTenant: null,
+		subscription: null,
+		location: null,
+		resourceGroup: null,
+		targetServerInstance: null,
+		migrationMode: null,
+		networkContainerType: null,
+		networkShares: [],
+		blobs: [],
+		targetDatabaseNames: [],
+		sqlMigrationService: undefined,
+		serverAssessment: null,
+		skuRecommendation: null,
+		serviceResourceGroup: null,
+		serviceSubscription: null,
+	};
+
+	if (assessmentReport.DmaVersion !== undefined) { //DMA assessment format import
+		switch (assessmentReport.TargetPlatform) {
+			case "VMSqlServer": {
+				saveInfo.migrationTargetType = MigrationTargetType.SQLVM;
+				break;
+			}
+			case "ManagedSqlServer": {
+				saveInfo.migrationTargetType = MigrationTargetType.SQLMI;
+				break;
+			}
+			case "AzureSqlDatabase": {
+				saveInfo.migrationTargetType = MigrationTargetType.SQLDB;
+				break;
+			}
+			default: {
+				if (assessmentReport.TargetPlatform.startsWith("SqlServer")) {
```

Comment:
I couldn't get this part, what are scenario we can hit the condition?

Reply:
removed